scripts-pipeline/fs.sh

```diff
@@ -68,9 +69,13 @@ this directory already exists. Delete it first and re-run the script, or pass
     fi
 fi
 
-if [ -n "${mask-}" ]; then
+# If mask is given, then mask the img
+if [ -f "${mask}" ]; then
     log "Mask t1"
-    run $SCRIPTDIR/mask "$t1" "$mask" "$t1"
+    run $SCRIPTDIR/mask $t1 $mask /tmp/t1masked.nii.gz
+    t1=/tmp/t1masked.nii.gz
```

Comment:
Let's look at the other usage of the mask script:
```
fs2dwi_T2.sh:run $SCRIPTDIR/mask "$t2" "$t2mask" $maskedt2
fs2dwi_T2.sh:run $SCRIPTDIR/mask "$t1" "$t1mask" $maskedt1
```
Given the above and also naturally, we don't want to overwrite the existing $t1. So, I suggest using-
`run $SCRIPTDIR/mask $t1 $mask /tmp/t1masked.nii.gz`

Reply:
Sounds good. But need to keep the quotation marks on all the arguments -- those are there in case the path has spaces in it.

Reply:
Done, thanks!